packages/docs/docs/sdk/Kotlin/android/io.logto.sdk.android.auth.logto/-logto-web-view-auth-activity/index.md

```diff
@@ -0,0 +1,419 @@
+# LogtoWebViewAuthActivity
+
+
+class [LogtoWebViewAuthActivity](index.md) : [AppCompatActivity](https://developer.android.com/reference/kotlin/androidx/appcompat/app/AppCompatActivity.html)
+
+## Constructors
+
+| | |
+|---|---|
+| [LogtoWebViewAuthActivity](-logto-web-view-auth-activity-constructor) | fun [LogtoWebViewAuthActivity](-logto-web-view-auth-activity-constructor)() |
+
+## Types
+
+| Name | Summary |
+|---|---|
+| [Companion](-companion/index.md) | object [Companion](-companion/index.md) |
+
+## Functions
+
+| Name | Summary |
+|---|---|
+| [addContentView](index.md#-652519453%2FFunctions%2F1239322798) | open override fun [addContentView](index.md#-652519453%2FFunctions%2F1239322798)(p0: [View](https://developer.android.com/reference/kotlin/android/view/View.html), p1: [ViewGroup.LayoutParams](https://developer.android.com/reference/kotlin/android/view/ViewGroup.LayoutParams.html)) |
+| [addOnContextAvailableListener](index.md#1898328035%2FFunctions%2F1239322798) | override fun [addOnContextAvailableListener](index.md#1898328035%2FFunctions%2F1239322798)(@[NonNull](https://developer.android.com/reference/kotlin/androidx/annotation/NonNull.html)p0: [OnContextAvailableListener](https://developer.android.com/reference/kotlin/androidx/activity/contextaware/OnContextAvailableListener.html)) |
+| [applyOverrideConfiguration](../../io.logto.sdk.android.auth.social.wechat/-wechat-social-result-activity/index.md#1950433422%2FFunctions%2F1239322798) | open fun [applyOverrideConfiguration](../../io.logto.sdk.android.auth.social.wechat/-wechat-social-result-activity/index.md#1950433422%2FFunctions%2F1239322798)(p0: [Configuration](https://developer.android.com/reference/kotlin/android/content/res/Configuration.html)) |
+| [attachBaseContext](index.md#-149173880%2FFunctions%2F1239322798) | open override fun [attachBaseContext](index.md#-149173880%2FFunctions%2F1239322798)(p0: [Context](https://developer.android.com/reference/kotlin/android/content/Context.html)) |
```

Comment:
no need for such detailed functions? looks like they are from the official android definition

Reply:
updated